.github/workflows/prod-deploy.yml

```diff
@@ -52,35 +52,29 @@ jobs:
           sudo rm -rf /opt/ghc
           sudo rm -rf "/usr/local/share/boost"
           sudo rm -rf "$AGENT_TOOLSDIRECTORY"
-      - name: Build Docker Image
-        run: bash scripts/build_docker_builder.sh prod $DOCKER_LABEL $PRIVATE_ECR $AWS_REGION
-        env:
-          DOCKER_LABEL: ${{ secrets.DOCKER_LABEL }}
-          REPO_NAME: ${{ secrets.PROD_REPO_NAME }}
-          AWS_REGION: ${{ secrets.AWS_REGION }}
-          ECR_URL: ${{ secrets.ECR_URL }}
-          PRIVATE_ECR: ${{ secrets.PRIVATE_ECR }}
-      - name: delete package.json
-        run: rm package.json
+      - name: move package.json
+        run: mv package.json package.jsonmoved
       - name: npm-install 'cli' and 'aws-sdk'
         run: npm install cli aws-sdk
-      - name: Publish to Elastic Container Registry
-        run: bash scripts/publish_ecr_builder.sh prod $GITHUB_SHA $DOCKER_LABEL $PRIVATE_ECR $AWS_REGION
+      - name: restore package.json
+        run: mv package.jsonmoved package.json
+      - name: Expose GitHub Runtime
+        uses: crazy-max/ghaction-github-runtime@v2
```

Comment:
This is used to access STORAGE_AWS_ACCESS_KEYs?

Reply:
It's actually to access some GitHub Actions environment variables used for caching Docker cache within GH Actions.